functions/check_prices/test_check_prices.py

```
@@ -0,0 +1,50 @@
+    assert status_code == 200
+    data = json.loads(response_body)
+    assert data['message'] == 'Price check completed'
+    assert 'timestamp' in data
+
+
+def test_check_prices_method_not_allowed():
+    """POSTメソッドは許可されない"""
+    request = Mock()
+    request.method = 'POST'
+    
+    response_body, status_code, headers = check_prices(request)
```

Ruff (0.14.2)

Unpacked variable `headers` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)